examples/experimental/i3s-17-and-debug/tile-debug.js

```diff
@@ -334,3 +315,80 @@ function isAllVerticesInsideBoundingVolume(boundingVolume, positions) {
 
   return isVerticesInsideObb;
 }
+
+export function isTileGeometryInsideBoundingVolume(tile) {
```

Comment:
No JSDoc/types?

No test cases?

Reply:
[Refactoring PR](https://github.com/visgl/loaders.gl/pull/1323)
It is not possible to implement tests outside of modules folder for now. We should investigate on that and also cover this code by tests.